README.md

```diff
@@ -70,7 +70,7 @@ All images are based on their official variants, being:
 
 ## Alpine Linux situation
 
-Even though both of the images are based on the Alpine Linux, the PHP official repository gives us the option to choose between its versions, at this moment being `3.7` or `3.8`.
+Even though both of the images are based on the Alpine Linux, the PHP official repository gives us the option to choose between its versions, at this moment being `3.8` or `3.9`.
```

Comment:
3.7 is still possible to use!

Reply:
It's no longer in the list of [supported tags](https://hub.docker.com/_/php?tab=description). Should we reconsider and perhaps drop them from the build matrix?

Reply:
Hmmm true, we still don't have a policy on cleaning up old tags, I think for now you could just create an issue for it and let's discuss what to do!

Reply:
> So what was the reason for the dots and new lines in the markdown?
> If it's the md linter I'd suggest to put it in the circleci config thus it won't happen again!

It wasn't any linter, but this just reads a lot more pleasantly regardless of your IDE settings :wink:

Reply:
Sounds good!